proposals/interface-types/working-notes/Instructions.md

```diff
@@ -0,0 +1,277 @@
+# Interface instructions
+
+This proposal defines new instructions that operate over interface value types. These instructions have the same form as core instructions; they push and pop values on an execution stack and support single pass validation.
+
+The new interface instructions are summarized below.
+
+```
+#interface-instr ::=
+  | sNN.from_iMM
+  | iMM.from_sNN
+
+  | uNN.from_iMM
+  | iMM.from_uNN
+
+  | string.size ..
+  | string.lift_memory ..
+  | string.lower_memory ..
+
+  | record.lift ..
+  | record.lower ..
+
+  | array.count
+  | array.lift ..
+  | array.lower ..
+
+  | invoke ..
+
+  | dup
```

Comment:
Could we instead say we're importing `let` from core wasm (the function-references extension)?  It's more general than `dup` and it seems unnecessarily asymmetric for adapter instructions to include `dup` if core wasm doesn't.

Reply:
Yes, I forgot we had discussed let. It's just let now.